scripts/globals/msg.lua

```diff
@@ -161,6 +161,7 @@ xi.msg.basic =
     STEAL_FAIL             = 153, -- <user> uses <ability>. <user> fails to steal from <target>.
     CANNOT_ON_THAT_TARG    = 155, -- You cannot perform that action on the specified target.
     REQUIRES_SHIELD        = 199, -- That action requires a shield.
+    IS_STATUS              = 84,  -- <target> is <status>.
```

Comment:
My OCD humbly requests this new addition be moved to an earlier line in the same catagory

Reply:
or else down to the existing category of status thingers

Reply:
oh, yeah, the 2nd thing - there's an identical message there, gonna need to alter name of one of them!
https://github.com/LandSandBoat/server/pull/2873/files#diff-962260141be49e3613198746d8b1c649c2aac2fc9955b6124cbeb092395af2edR260